command/issue.go

```diff
@@ -397,7 +397,7 @@ func issueCreate(cmd *cobra.Command, args []string) error {
 				url.QueryEscape(title),
 				url.QueryEscape(body),
 			)
-		} else if len(templateFiles) > 1 {
+		} else if len(nonLegacyTemplateFiles) > 1 {
```

Comment:
The /choose route is to be opened only if we have non-legacy templates. If there is a legacy template only, it is selected by default by GitHub